db/migrate/20180928185043_add_locale_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddLocaleToUser < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :locale, :string, null:false
```

Comment:
Add a default to this, if the user doesn't set locale, set it to en, otherwise is gonna fail when setting de I18n

Reply:
Nevermind, i see you set a locale in the I18n config